services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -199,14 +200,20 @@ async def _start_dynamic_service(
 ):
     if not _is_node_dynamic(service_key):
         return
-    project_running_nodes = await director_v2_api.list_dynamic_services(
+    project_running_nodes: list[DataType] = await director_v2_api.list_dynamic_services(
         request.app, user_id, f"{project_uuid}"
     )
 
+    project_active_running_nodes = [
+        node
+        for node in project_running_nodes
+        if node["service_state"] in {"pending", "pulling", "starting", "running"}
```

Comment:
Maybe also pull out the set of "considered running/starting states" an a module constant.